components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -606,6 +606,28 @@
   width: 48%;
   float: right;
 }
+.menu-box .field-deceasedAge {
+  width: 47%;
+  float: left;
+}
+.menu-box .field-deceasedCause {
+  width: 48%;
```

Comment:
by the same reason `.menu-box .field-first_name` has `width: 47%` and `.menu-box .field-last_name` has `width: 48%`. Just fitting additional fields into the existing layout.